src/main/scala/chisel3/Driver.scala

```diff
@@ -241,10 +241,21 @@ object Driver extends BackendCompilationUtilities {
     val annosx = try {
       phases.foldLeft(annos)( (a, p) => p.transform(a) )
     } catch {
-      /* ChiselStage and FirrtlStage can throw StageError. Since Driver is not a StageMain, it cannot catch these. While
-       * Driver is deprecated and removed in 3.2.1+, the Driver catches all errors.
+      /* Catch all errors and show a fake stack trace. If the stack trace was minified elsewhere, e.g., in the Elaborate
+       * phase, then it will show up minified here. The Driver cannot fail for any reason. Instead, a
+       * ChiselExecutionFailure result will show up when trying to view the resulting annotation sequence.
        */
-      case e: StageError => annos
+      case a: Throwable =>
+        val fakeStackTrace =
+          Predef
+            .augmentString(
+              s"""|\t...
+                  |${a.getStackTrace().view.mkString("\tat ", "\n\tat ", "")}
+                  |""".stripMargin)
```

Comment:
I think these extra ellipses are unnecessary since we have them inside the stack trace now, it prints something like:
```
Elaborating design...
[error]         ...
[error]         at ... ()
[error]         at gcd.GCD.$anonfun$new$3(GCD.scala:24)
[error]         at chisel3.Data.$anonfun$$colon$eq$1(Data.scala:524)
[error]         at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[error]         at chisel3.internal.prefix$.apply(prefix.scala:32)
[error]         at chisel3.Data.$colon$eq(Data.scala:524)
[error]         at gcd.GCD.$anonfun$new$2(GCD.scala:24)
[error]         at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[error]         at chisel3.WhenContext.<init>(When.scala:124)
[error]         at chisel3.when$.apply(When.scala:31)
[error]         at gcd.GCD.<init>(GCD.scala:24)
[error]         at gcd.GCDMain$.$anonfun$new$12(GCD.scala:40)
[error]         at ... ()
[error]         at ... (Stack trace trimmed to user code only. Rerun with --full-stacktrace to see the full stack trace)   
```

Reply:
Will fix in rebase.